tests/test_falconparser.py

```diff
@@ -42,3 +43,10 @@ def test_invalid_json(self, testapp):
     def test_parsing_headers(self, testapp):
         res = testapp.get("/echo_headers", headers={"name": "Fred"})
         assert res.json == {"NAME": "Fred"}
+
+    # `falcon.testing.TestClient.simulate_request` parses request with `wsgiref`
+    def test_body_parsing_works_with_simulate(self, testapp):
```

Comment:
I don't think testapp is needed, here. Is it?

Reply:
You're right, it's not - updated.